UserSettings/EditorUserSettings.asset

```diff
@@ -0,0 +1,28 @@
+%YAML 1.1
```

Comment:
Correct me if im wrong, but is is normal to add UserSettings to git? I've never worked with unity and git before so im not sure myself, but it seams like the kind of file that would end up causing a lot of merge conflicts

Reply:
I'll leave it out for now. Adding to gitignore in [this pr](https://github.com/VildeStabell/master-thesis-game/pull/2)